app/models/user.rb

```diff
@@ -10,6 +10,14 @@ class User < ApplicationRecord
   has_many :completed_lessons, through: :lesson_completions, source: :lesson
   has_many :projects
 
+  def format_completion_date
```

Comment:
How about refactoring this method to 

```ruby
def format_completion_date
  learning_goal_completion_date&.strftime('%B %Y') ||
  'Click here to set a date!'
end
```

Ignore if my version isn't readable.